modules/aws/ssm.go

```diff
@@ -0,0 +1,72 @@
+package aws
+
+import (
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ssm"
+)
+
+func GetParameter(t *testing.T, awsRegion string, keyName string) string {
+	keyValue, err := GetParameterE(t, awsRegion, keyName)
+	if err != nil {
+		t.Fatal(err)
+	}
```

Comment:
We've started shortening this to a one liner: `require.NoError(t, err)`

Reply:
I have updated PR to use the recommended pattern.